src/category_theory/elements.lean

```diff
@@ -1,17 +1,46 @@
-import category_theory.types
-import category_theory.comma
+/-
+-- Copyright (c) 2019 Scott Morrison. All rights reserved.
```

Comment:
Line comments are unnecessary now.

Reply:
Okay, I fixed these, and also every other instance I could find. Sorry, I'd never realised I was the sole person using line comments in the copyright headers! (With a few exceptions, typically my students.)